interpreter/src/main/scala/pl/touk/nussknacker/engine/compile/ComponentExecutorFactory.scala

```diff
@@ -7,25 +7,26 @@ import pl.touk.nussknacker.engine.api.context.ProcessCompilationError
 import pl.touk.nussknacker.engine.api.definition.Parameter
 import pl.touk.nussknacker.engine.api.process.ComponentUseCase
 import pl.touk.nussknacker.engine.compile.nodecompilation.ParameterEvaluator
-import pl.touk.nussknacker.engine.compiledgraph.evaluatedparam
-import pl.touk.nussknacker.engine.compiledgraph.evaluatedparam.TypedParameter
+import pl.touk.nussknacker.engine.compiledgraph.TypedParameter
 import pl.touk.nussknacker.engine.definition.component.ComponentDefinitionWithImplementation
 import pl.touk.nussknacker.engine.expression.ExpressionEvaluator
 
-class FactoryComponentInvoker(expressionEvaluator: ExpressionEvaluator) extends LazyLogging {
+// This class helps to create an Executor using Component. Most Components are just a factories that creates "Executors".
+// The situation is different for non-eager Services where Component is an Executor, so invokeMethod is run for each request
+class ComponentExecutorFactory(expressionEvaluator: ExpressionEvaluator) extends LazyLogging {
```

Comment:
the PR description is outdated (FactoryComponentInvoker => ComponentProcessingObjectFactory)

Reply:
also StubbedFragmentInputDefinitionSource => StubbedFragmentSourceDefinitionPreparer seems to be outdated

Reply:
Good catch. In the first case, description was outdated, in the second, the implementation was not changed - fixed